index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Blue Recycling</title>
+    <link rel="stylesheet" href="styling.css">
+</head>
+
+<body>
+    <div class="wrapper-sections">
+        <div class='section' id='newsletter'>
+            <div class="tablet">
+                <span id='join'>Join our</span>
```

Comment:
Maybe a bit generic id's? Could be something like joinSpan and newsSpan or something similar